horovod/tensorflow/keras/__init__.py

```diff
@@ -63,10 +64,15 @@ def DistributedOptimizer(optimizer, name=None,
                          the original sparse gradient has high density.
                          Defaults to false.
         aggregation_frequency: How many batches to aggregate the gradients before
-                               averaging the gradients with allreduce.      """
+                               averaging the gradients with allreduce.
+
+        grad_updated_sizes_dict: A dictionary containing the shape of each
+        update grad.
```

Comment:
for consistency the `update grad` should be aligned with `A dictionary`

Reply:
Done.